### PR TITLE
Update dev dependencies and increase version to 0.0.4

### PR DESCRIPTION
**Pull request by SupremeVoid**
As requested. Increased version and updated dev dependencies in a separate PR.

Iincreased the version to 0.0.4-0 since the current released package is 0.0.3-0.

https://github.com/typed-mxgraph/typed-mxgraph/pull/5

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@typed-mxgraph/typed-mxgraph",
-  "version": "0.0.2-0",
+  "version": "0.0.4-0",
```
Comment (tbouffard) [inline package.json:3]: In `mxgraph-type-definitiions`, we used to store the latest release version until a new release is done.
I don't know what we want do it for the `typed-mxgraph` package.
For sure, we cannot keep `0.0.2-0` as `0.0.3-0` is already available in npmjs

Comment (tbouffard) [reply package.json:3]: cc @hungtcs

**Comment by hungtcs** [comment]
I am using `npm version prepatch` to increase version number and auto tag, so I will close this PR.

**Comment by hungtcs** [comment]
New version has been released

**Pull request closed without merging**